### DISCUSSION TITLE
Linux Agent - Network PID Discover first time Run

### DISCUSSION THREAD

**Discussion by dkleckner498** [amidaware/tacticalrmm#1037] [Ideas]
Sorry for all of these idea requests but figured I'd throw them out there.  I have seen this done before but when an agent first gets added to a Linux OS to have it do an auto discover of current running PIDs and then auto create Check Scripts for those PIDS to see if they are running.  Windows I have seen similar but it ends up being too much noise and is better off creating the scripts checks at a global level.  Linux on the other hand usually runs very unique and sometimes specific applications.  One example is with 3CX and adding the RMM Agent, it would do auto discover of running PIDs and if there is anything you don't want to monitor you can remove the check since it only runs the first time during install.

**Comment by silversword411** [comment]
Discussions will get lost...if it's a new non-duplicate enhancement request create a Github Issue for tracking that
